docker/server/Dockerfile

```diff
@@ -68,8 +67,14 @@ RUN groupadd -r clickhouse --gid=101 \
                 clickhouse-client=$version \
                 clickhouse-server=$version ; \
        fi \
-    && wget --progress=bar:force:noscroll "https://github.com/tianon/gosu/releases/download/$gosu_ver/gosu-$(dpkg --print-architecture)" -O /bin/gosu \
-    && chmod +x /bin/gosu \
+    && wget --progress=bar:force:noscroll -O /usr/local/bin/su-exec.c https://raw.githubusercontent.com/ncopa/su-exec/master/su-exec.c && \
```

Comment:
The code looks quite simple. Maybe copy-paste code here to avoid supply chain attacks
(someone may hijack the GitHub repo and we may leave it unnoticed during Docker image rebuild).

Or, more simple: replace master with commit sha
(so we will only trust GitHub, SHA1, wget, root certificates, gcc, Linux kernel and AWS machines).

Reply:
I'll look into that.